packages/components/combobox/src/ComboboxItems.tsx

```diff
@@ -29,8 +29,8 @@ export const Items = forwardRef(
         ref={ref}
         className={cx(
           className,
-          'relative flex flex-col',
-          isOpen ? 'block' : 'hidden',
+          'flex flex-col',
+          isOpen ? 'block' : 'pointer-events-none invisible opacity-0',
```

Comment:
`invisible` instead of `hidden` preserves the dimensions and helps radix calculates the position more efficiently.